### PR TITLE
add: inputs to logging for tool

### PR DESCRIPTION
**Pull request by douglas-reid**
Added some more for context on what was being sent to individual tools.

### CODE REVIEW

**Review by eob — APPROVED** [review]
These all look like good changes to me

File: src/steamship/agents/service/agent_service.py [hunk 1]
```diff
@@ -29,15 +29,26 @@ def run_action(self, action: Action, context: AgentContext):
...
             context.completed_steps.append(action)
 
     def run_agent(self, agent: Agent, context: AgentContext):
         action = agent.next_action(context=context)
         while not isinstance(action, FinishAction):
             # TODO: Arrive at a solid design for the details of this structured log object
+            inputs = ",".join([f"{b.as_llm_input()}" for b in action.input])
             logging.info(
-                f"Running Tool {action.tool.name}",
+                f"Running Tool {action.tool.name} ({inputs})",
```
Comment (dkolas) [inline src/steamship/agents/service/agent_service.py:51]: Does this push user data to our log system?

**Review by dkolas — APPROVED** [review]
Great fixes! One thing - we may not want to log inputs / outputs at least at the `info` level, as that will store them in Elastic. We can set those to `debug` or come up with another strategy.

File: src/steamship/agents/service/agent_service.py (hunk 1 above)
Comment (douglas-reid) [reply src/steamship/agents/service/agent_service.py:51]: potentially. the text of whatever the agent looks at, at least. so, when a tool is called with user input directly, then yes?

**Pull request merged**